Array_Control/Array_Control.ino

```diff
@@ -42,10 +40,10 @@ bool DEBUG = true;
 #define UNLOADED_VOLTAGE_MES_WAIT_TIME //ms
 
 //MOSFET switching times (seconds)
-#define MOSFET_TURN_ON   0.000000021 
-#define MOSFET_RISE_TIME 0.000000120 
-#define MOSFET_TURN_OFF  0.000000180 
-#define MOSFET_FALL_TIME 0.000000115
+#define MOSFET_TURN_ON   0.000000021f 
+#define MOSFET_RISE_TIME 0.000000120f 
+#define MOSFET_TURN_OFF  0.000000180f 
```

Comment:
why the "f"?

Reply:
the `f` makes it a floating point instead of a double. Allows for better accuracy.